sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -77,10 +77,22 @@ static int connect_device();
 static int subscribe();
 
 //
-// Messaging functions
+// Device registration functions
 //
 static int register_device();
 static int get_operation_status();
+static az_result parse_operation_message(
+    char* topic,
+    int topic_len,
+    MQTTClient_message* message,
+    az_iot_provisioning_client_register_response* response,
+    az_iot_provisioning_client_operation_status* operation_status);
+static int send_operation_query_message(az_iot_provisioning_client_register_response response);
+
+//
```

Comment:
Nit: I don't like "helper" or "util" or any other generic labeling: they are not adding any new information and are obscure enough to be a dumping ground for all functions (especially in OOP).

Reply:
Do you have a suggestion for how else to group these?  They aren't directly registration functions, and I think they crowd the space of that grouping.

Reply:
I don't think these can be grouped.